help/domains/bio/bioinformatics.md

```diff
@@ -0,0 +1,406 @@
+associated with sequences.
+
+For identifiers that do not fit in a cell, an ellipsis is shown.
+
+![Visualize Sequence Alignments](../../uploads/macromolecules/macromolecules-6.png "Visualize Sequence Alignments")
+
+Sequence alignment visualization allows for manually modifying the alignment in the case when automatic alignment result
+is not satisfactory. The composition analysis plot changes interactively, providing a quick indication of how good the
+alignment is.
+
+![Manual alignment by editing sequence in free text](../../uploads/macromolecules/macromolecules-1.gif "Manual alignment by editing sequence in free text")
+
+-->
+
+## Data editing
+
+Datagrok offers custom data editing capabilities directly within the dataframe for certain data types:
+
+* For DNA, RNA, and protein sequences in FASTA-like format, you can edit the sequences as needed.
+* If you have loaded the HELM monomer library, you can edit HELM sequences.
+This includes adding or removing monomers and modifying connections.
+The editor supports circular and branching structures.
+
+![HELM editor](img/HELM_editor.png)
+
+## Data filtering
+
+When working with a sequence column, you can specify a subsequence
+and filter the dataframe to display only sequences that contain this specified subsequence.
+
+The filtering is performed based on a 100% exact match of the subsequence.
+Datagrok does not currently support subsequence filtering with mismatches.
+
+## Macromolecule-specific viewers and actions
+
+Specially for analyzing macromolecules,
+Datagrok provides a range of specialized viewers and analysis tools,
+as well as  number of tools and applications specifically for polymer modalities.
+You can find most of them in the **Bio** section of the main menu.
+
+In addition, out of the box Datagrok provides a comprehensive
+[machine learning toolkit](../../learn/data-science.md) for
+clustering, dimensionality reduction techniques, imputation, PCA/PLS, etc.
+Some of these tools could be directly applied
+to the polymer modalities (for instance, by mapping monomers to features)
+and used for analyzing structure-property,
+structure-activity, and sequence-activity relationships.
+
+### Multiple Sequence Alignment (MSA)
+
+The [Multiple Sequence Alignment](https://en.wikipedia.org/wiki/Multiple_sequence_alignment)
+(MSA) feature allows you to create alignments of biological sequences.
+This action generates a new column containing the aligned sequences.
+MSA works for macromolecules in both FASTA and HELM formats.
+
+If your data has already been clustered, Datagrok provides an option to specify a "cluster" column
+to align sequences only within the same cluster.
+
+![Multiple Sequence Alignment dialog](img/MSA_dialog.png)
+
+For DNA, RNA, and natural peptides, Datagrok utilizes
+[KAlign](https://github.com/TimoLassmann/kalign) to perform the alignment.
+It is an open-source tool under GNU GPL, so it can be modified
```

Comment:
It is an open-source tool under GNU GPL, so it can be modified
to work with custom substitution matrices for
sequences. - remove